### ISSUE TITLE
Wrong network adapter is used by Mira Web (Win 7)

### ISSUE THREAD

**Issue by 11ols** [Cycling74/miraweb#72]
Hi!
Details:
on windows 7 64bit I have some virtual network adapters installed. additionally to my default wlan adapter. Mira(web) is using one of the virtual adapters! (i assume it picks the last adapter in the list)
I can access the ip on the same computer but of course not with other clients in my wlan. I add a pic, which makes it very clear.

The Problem is I can't tell mira which adapter to use and I wonder why it does not use the default wlan adapter.
![mira_adapter_pick](https://cloud.githubusercontent.com/assets/21139297/24291874/20892dcc-108b-11e7-920a-eb0d036cfc89.png)

cheers,
Oliver

**Comment by benbrackenc74** [comment]
Deferring this for a future release.

**Comment by 1offby** [comment]
I'm running into this as well, as MiraWeb is finding a client's intranet Ethernet adapter rather than the WiFi adapter needed for finding a tablet.

**Comment by 1offby** [comment]
Just a use-case update here:
Was using iPad app in an effort to improve Mira graphics performance (#79, #58). Gave up. Stripped that stuff out of the Mira parts of this project. 
iPad app does find mira.frame in the cases where there's more than one active ethernet adapter.
Now hope to revert to MiraWeb WiFi, 'cause I'm a believer!!
But client machines have 2 active ethernet adapters
1) Wired-only-non-internet-connected-intranet (useless for WiFi MiraWeb)
2) WiFi card (in this case AirPort).
MiraWeb socket only finds (1)
So, I'm cornered into using the Mira App...

**Comment by fde31** [comment]
Just a heads up that this might only be related to the built-in reporting of MiraWeb while the server/client might/should still be available. I agree that it's not ideal and we'll try to look into this soon.

In the meantime could you do me favour and let me know if you can access MiraWeb in the described cases by ignoring the reported IP address and instead look up the address manually and access the MW client using that. It would help us distinguishing a bit further what sort of work is necessary in order to tackle this problem. Does that make sense for your use-case @1offby ? Here the simple steps:

```
1. Run MW enabled patcher
2. Ignore reported IP address in Max console/mira.status and instead lookup machine's IP manually
3. Try accessing MW using the IP address from _2_ while sticking to the port reported by Max

=> http://<manual_looked_up_ip>:<port_given_in_max>
```

Thanks a bunch in advance!

**Comment by 1offby** [comment]
I'll try this ASAP.
This potential workaround makes intuitive sense.
It'll be a couple days before I'm at a location with this type setup.
Thank you!

**Comment by fde31** [comment]
any update on this @1offby? Thanks for the help!

**Comment by 1offby** [comment]
No, the systems where I should test this are currently in full time use. Probably be able to check this in the 1st/2nd week of September.
Sorry for the delay...

**Comment by 1offby** [comment]
Yup.
Substituting the desired adapter's IP address in a browser connected to the same network does indeed open the MiraWeb connection, despite a different address being reported in Miraweb's Max websocket  notification.
Very good!

**Comment by starakaj** [comment]
I wish this was an easy fix, but it's gonna require changes to libXebra itself I think.
